java-frontend/src/test/files/se/ConditionAlwaysTrueOrFalseWithoutJSR305.java

```diff
@@ -0,0 +1,25 @@
+package my.app;
+
+import org.springframework.web.context.request.RequestAttributes;
```

Comment:
Name of the file is a bit dodgy : this is only testing spring annotations via the rule and not the rule : I would not use the ConditionAlwaysTrueOrFalse in the name of the file.

Reply:
renamed